cmd/argocd/commands/app.go

```diff
@@ -1125,6 +1125,8 @@ func NewApplicationDiffCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 		serverSideGenerate bool
 		localIncludes      []string
 		appNamespace       string
+		revisions          []string
+		sourceIndexes      []int64
```

Comment:
Should this be `int64` ? I feel we can use `uint` or `unit32` here as the index positions do not have negative numbers.

Reply:
The command does not support `uint` or `uint32`. Thus, had to keep it `int` or `int64`. Had used `int64` to keep it consistent with what is needed by `revisionSourceMappings`. I can change this to `int` and then typecast it to `int64` for creating `revisionSourceMappings`.

Reply:
If flagset support is missing for `uint`, then let's stick to `int64`